src/components/SignupForm/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+
+import logo from '../../assets/logo.png';
```

Comment:
assets should be on `src/app/assets` not on `src` folder, its an advice for future problems ;)